src/samples/audio/Kconfig

```diff
@@ -11,7 +11,7 @@ menu "Audio component samples"
 	                Select for test smart amplifier component
 
         config SAMPLE_KEYPHRASE
-		depends on CAVS
+		depends on CAVS || IMX
```

Comment:
Commit message is slightly inaccurate. You are adding a dependence on the IMX platform, not architecture.

I'd rephrase the commit message as "Add IMX platform as a dependency for SAMPLE_KEYPHRASE" and a second line "This allows i.MX based platforms to use the samples for the keyphrase detector".